futures-util/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 //! and the `AsyncRead` and `AsyncWrite` traits.
 
 #![cfg_attr(feature = "alloc", feature(box_into_pin))]
```

Comment:
`box_into_pin` is also an unstable feature.

Reply:
😲 why is this not failing in travis then.

Oh, `allow-features` is only for language features. Is there any way to deny library features as well?

Reply:
No, I don't believe so-- seems like a reasonable thing to add to `allow-features` (cc @tmandry).

Reply:
Opened https://github.com/rust-lang/rust/pull/60289